modules/nextMillenniumBidAdapter.js

```diff
@@ -81,65 +81,46 @@ export const spec = {
   buildRequests: function(validBidRequests, bidderRequest) {
     const requests = [];
     window.nmmRefreshCounts = window.nmmRefreshCounts || {};
+    const site = getSiteObj();
+    const device = getDeviceObj();
```

Comment:
all the contents of this function can be replaced with examining the request, i believe the same is true for the getSiteObj function

Reply:
I specifically moved this part of the code into separate functions so that each part of the openrtb request could be collected separately.